bibtexparser/bwriter.py

```diff
@@ -14,6 +16,23 @@
 __all__ = ['BibTexWriter']
 
 
+class SortingStrategy(Enum):
```

Comment:
I think we need some docstrings on this enum. There seems to be no universal agreement on how to do that - I'd propose to go along with the following: https://stackoverflow.com/a/72775639

Reply:
👍 . I've added a docstring, but I'm not sure about whether this docstring will be processed properly: 
```
:py:attr:`~.BibTexWriter.display_order`
```

PyCharm doesn't render it..

Reply:
Yeh. Enum/Attribute docstrings are quite a hassle. And it's not getting better when you combine them. 

Either way, I think the current version is as good as we can do. Should future python versions define some better syntax, we can quickly change it. Thanks!